lib/TBDGen/TBDGen.cpp

```diff
@@ -90,6 +90,11 @@ class TBDGenVisitor : public ASTVisitor<TBDGenVisitor> {
       return;
 
     auto declRef = SILDeclRef(VD);
+
+    // Transparent symbols don't exist, even if public.
+    if (declRef.isTransparent())
```

Comment:
The only `isSILOnly` method I can find is on `LinkEntity`, and it's not obvious to me how to get one of those from this, e.g. the `LinkEntity::Kind::Function` doesn't ever seem to be used (no constructor, etc.), and the `SILFunction` kind can't work.

I obviously can just add the infrastructure required to work with the `Kind::Function`s, but that seems a little weird.